src/content/smart-bulletins/031.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Smart Bulletin No. 031"
+intro: "GSA SmartPay – OMB Circular A-123, Appendix B Implementing Guidance and Clarifications/Updates"
+number: "No. 031"
+order: 031
+businessline: "Purchase, Travel, Fleet and Integrated"
+published: "Dec 19, 2019"
+lastupdated: "N/A"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| December 19, 2019 | Implementation of Smart Bulletin #31 |
+
+## Effective Date
+
+Effective upon issuance and shall remain in force until modified,
+rescinded, or superseded.
+
+## Business Line(s) Affected
+
+Purchase, Travel, Fleet and Integrated
+
+## Introduction
+
+The purpose of this GSA SmartPay Smart Bulletin is to provide implementation guidance, updates and clarifications relating to portions of OMB Circular A-123, Appendix B, ‘A Risk Framework for Charge Card Programs (“the Circular”). These revisions were coordinated with the Office of Management and Budget, Office of Federal Financial Management.
+
+## Summary
+
+This Smart Bulletin addresses issues regarding creditworthiness, sustainability, environmental sustainability, definition of terms and other matters.
+
+### Chapter 5 – Performance Metrics and Data Requirements
+Section 5.3.1 implementing statistical reporting is updated as follows: 
+
+The reference in this section to the GSA SmartPay website shall be read to refer to the GSA Interact, the secure platform used to post agency-specific metrics.
+### Chapter 6 – Credit Worthiness for Individually Billed Accounts (IBAs)
+Section 6.4 implementing guidance is updated as follows: 
```

Comment:
Section 6.4 implementing guidance is updated as follows:
If it is not possible to obtain a card applicant’s credit score, the agency may issue a “restricted” charge card (as defined in section 6.3.2.,” First time travel charge card applicants with a credit score of less than 660”) to a first time applicant, but the agency must ensure it first conducts an alternative creditworthiness assessment to determine whether the individual possesses a satisfactory credit history. Specifically, the agency must review the applicant’s most recent Standard Form (SF) 85P, Questionnaire for Public Trust Positions, Section 22, or SF 86, Section 26, or use a similar vehicle containing the same type of questions as in the forms and sections noted and use the information provided to assess creditworthiness. In either case, the vehicle used must not be older than one year.
Note: Neither the SF 86 nor SF 26 is to be shared with your servicing contractor bank.
Section 6.11 is clarified as follows:
The reference to the “OPM Center for Federal Investigative Services (FIS)” is hereby deleted and replaced with “Defense Counterintelligence and Security Agency (DCSA).”
Section 6.12 is clarified as follows:
The reference to GSA’s Financial and Business Solutions (FABS) Schedule contractors and its associated footnote is hereby deleted and replaced with “PS-MAS.” More information on PS-MAS (Professional Services – Multiple Award Schedule) in this context can be found at: gsa.gov/psschedules, under “professional services/identity protection services.”